### PR TITLE
Group buttons functionality fixed

### PR DESCRIPTION
**Pull request by squeakypickles33**

### CODE REVIEW

File: src/components/Group/GroupsElements.tsx [hunk 1]
```diff
@@ -13,13 +13,14 @@ export default function GroupsElements() {
...
               href={`/${group.id}`}
-              className="flex items-center gap-3">
+              className="flex flex-none items-center gap-3 ">
                 <Image src={`${group.photoUrl}`}
                 alt={`Image for ${group.name}`}
                 width={50}
                 height={50}
                 unoptimized={true}
-                className="relative h-10 w-10 rounded-lg bg-secondary ring-1 ring-base-200"/>
+                style={{ objectFit: "cover" }}
```
Comment (redconOne) [inline src/components/Group/GroupsElements.tsx]: I think we could handle the objectFit using [tailwind](https://tailwindcss.com/docs/object-fit). If this doesn't work or there's a specific reason that we're using inline-styling just let me know.

**Review by redconOne — CHANGES_REQUESTED** [review]

File: src/components/Group/GroupsElements.tsx (hunk 1 above)
Comment (squeakypickles33) [reply src/components/Group/GroupsElements.tsx]: hey just put that in as tailwind and not inline, my bad

Comment (redconOne) [reply src/components/Group/GroupsElements.tsx]: Thanks!

**Review by redconOne — APPROVED** [review]
Looks good!

**Pull request merged**